platform/app/prisma/migrations/20260805120000_fan_out_batch_and_variant/migration.sql

```
@@ -0,0 +1,54 @@
+    "status" "FanOutVariantStatus" NOT NULL DEFAULT 'PENDING',
+    "scenarioRunId" TEXT,
+    "decidedById" TEXT,
+    "decidedAt" TIMESTAMP(3),
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "FanOutVariant_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE INDEX "FanOutBatch_projectId_status_idx" ON "FanOutBatch"("projectId", "status");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)